IBPSA/Fluid/Chillers/BaseClasses/Carnot.mo

```diff
@@ -50,6 +50,14 @@ partial model Carnot
     "Pressure difference over evaporator"
     annotation (Dialog(group="Nominal condition"));
 
+  parameter Modelica.SIunits.TemperatureDifference TAppCon_nominal(min=0) = if cp1_default < 1500 then 5 else 2
+    "Temperature difference between refrigerant and working fluid outlet in condenser"
+    annotation (Dialog(group="Efficiency"));
+
+  parameter Modelica.SIunits.TemperatureDifference TAppEva_nominal(min=0) = if cp2_default < 1500 then 5 else 2
+    "Temperature difference between refrigerant and working fluid outlet in evaporator"
```

Comment:
It would make sense to let the name reflect that the parameter is a temperature difference?